app/models/occupation_standard.rb

```diff
@@ -12,6 +12,28 @@ class OccupationStandard < ApplicationRecord
 
   validates :title, presence: true
 
+  def unregistered_clone(creator_id:, organization_id:)
+    begin
+      OccupationStandard.transaction do
+        os = UnregisteredStandard.create!(
+          creator_id: creator_id,
+          organization_id: organization_id,
+          occupation: occupation,
+          title: title,
```

Comment:
I think it'd be worth using "#{title} COPY" or something of that nature. If it's the same title and same organization, the two standards could be hard to distinguish from one another. This is what we do on workhands, and then the user can edit afterwards.